clang/lib/Sema/SemaDeclCXX.cpp

```diff
@@ -16983,7 +16983,7 @@ VarDecl *Sema::BuildExceptionDeclaration(Scope *S,
 
 /// ActOnExceptionDeclarator - Parsed the exception-declarator in a C++ catch
 /// handler.
-Decl *Sema::ActOnExceptionDeclarator(Scope *S, Declarator &D) {
+Decl *Sema::ActOnExceptionDeclarator(Scope *S, Declarator &D, bool isCatchAll) {
```

Comment:
I am hesitate on this. IIUC, previously the `catch(...)` format is reprensented as the exception decl as nullptr? Then if yes, we should be able to reuse that.

Reply:
I think it won't be possible since if you see the ActOnExceptionDeclarator , we are not passing the Decl*, later we are updating the Decl*